src/actions/wizard.ts

```diff
@@ -0,0 +1,68 @@
+export const UPDATE_CURRENT_ACTION = "UPDATE_CURRENT_ACTION";
+export const UPDATE_FLASHING_PROPERTIES = "UPDATE_FLASHING_PROPERTIES";
+export const UPDATE_INSTALL_TYPE = "UPDATE_INSTALL_TYPE";
+export const UPDATE_SHOW_QUESTIONNAIRE = "UPDATE_SHOW_QUESTIONNAIRE";
+
+export enum WizardAction {
+  CONNECT = 1,
+  SETUP = 2,
+  FLASH = 3,
+  DONE = 4,
+}
+
+export interface FlashingProperties {
+  ssid: string;
+  password: string;
+  firmware: string;
```

Comment:
should we call this `firmware_version` ?

Reply:
si